src/sentry/monitors/processing_errors/manager.py

```diff
@@ -94,6 +94,30 @@ def _delete_for_entity(entity_identifier: str, uuid: uuid.UUID) -> None:
     pipeline.execute()
 
 
+def _delete_for_entity_by_type(entity_identifier: str, type: ProcessingErrorType) -> None:
+    checkin_errors = _get_for_entities([entity_identifier])
+    redis = _get_cluster()
+    pipeline = redis.pipeline()
+    for checkin_error in checkin_errors:
+        errors = checkin_error.errors
+        if not any([error["type"] == type for error in errors]):
```

Comment:
```suggestion
        if not any(error["type"] == type for error in errors):
```

nit, you don't need the list here, can just pass the generator directly to `any`

Reply:
oh very nice